.github/workflows/push-tag-release.yml

```diff
@@ -1,69 +1,263 @@
+        shell: bash
+        # TODO: Remove --from_source when all libraries are published to pypi
+        run: |
+          pip install -r tools/requirements.txt
+          python tools/build.py --remove_intermediate_folder --from_source build_connect_plugin ${{ needs.set-variables-create-release.outputs.folder }}
+      - name: build nuke studio connect plugin
+        if: ${{ needs.set-variables-create-release.outputs.package == 'nuke-studio' }}
+        shell: bash
+        # TODO: Remove --from_source when all libraries are published to pypi
+        run: |
+          pip install -r tools/requirements.txt
+          python tools/build.py --remove_intermediate_folder --from_source --include_resources resource/plugin,resource/application_hook build_connect_plugin ${{ needs.set-variables-create-release.outputs.folder }}
+      - name: build rv connect plugin
+        if: ${{ needs.set-variables-create-release.outputs.package == 'rv' }}
+        shell: bash
+        # TODO: Remove --from_source when all libraries are published to pypi
+        run: |
+          cd projects/rv
+          python build_rv_plugin.py resource/plugin /tmp/
+          cd ../..
+          pip install -r tools/requirements.txt
+          python tools/build.py --remove_intermediate_folder --from_source --include_assets /tmp/ftrack-5.1.0.rvpkg build_connect_plugin ${{ needs.set-variables-create-release.outputs.folder }}
+      - name: upload artifacts to release
+        shell: bash
+        run: |
+          cd ${{ needs.set-variables-create-release.outputs.folder }}
+          gh release upload ${{ needs.set-variables-create-release.outputs.tag }} dist/* --clobber
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: activate release
+        shell: bash
+        run: |
+          gh release edit ${{ needs.set-variables-create-release.outputs.tag }} --draft=false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload temp artifacts
+        uses: actions/upload-artifact@v2
+        if: ${{ needs.set-variables-create-release.outputs.to_pypi == 'true' }}
+        with:
+          name: dist
+          path: ${{ needs.set-variables-create-release.outputs.folder }}/dist
 
-  # Also align latest rc pre-release build to tag, as the previous push to main
-  # probably was just a release note update which is not picked up by Pants/merge CI.
+  build-windows:
+    needs: set-variables-create-release
+    concurrency:
+      group: ci-${{ github.ref }}
+    if: ${{ needs.set-variables-create-release.outputs.package == 'framework-photoshop' }}
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: install poetry
+        shell: bash
+        run: |
+          pip install poetry
+      - name: build style
+        if: ${{ needs.set-variables-create-release.outputs.package == 'qt-style' }}
```

Comment:
Shouldn't this be build only when releasing qt-style? I might be wrong, but then if we have to build it every time we release PS, we should download it from the published library qt-style right?

Reply:
It is only built if it is the qt-style yes, notice the ==

Reply:
It has to be removed form here as it has the if PS package before